<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Passages</class>
 <widget class="QDialog" name="Passages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 81, 100);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-bottom-left-radius:15px</string>
   </property>
   <property name="text">
    <string>I     D </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-bottom-left-radius:15px</string>
   </property>
   <property name="text">
    <string>姓    名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-bottom-left-radius:15px</string>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:15px;
background-color: rgb(255, 255, 255);
border-bottom-right-radius:15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:15px;
background-color: rgb(255, 255, 255);
border-bottom-right-radius:15px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:15px;
background-color: rgb(255, 255, 255);
border-bottom-right-radius:15px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton:!hover
{
background-color: rgb(255, 255, 255);
border-radius:8px;
}
#pushButton:hover
{
background-color: rgb(85, 187, 138);
border-radius:8px;
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2:!hover
{
background-color: rgb(255, 255, 255);
border-radius:8px;
}
#pushButton_2:hover
{
background-color: rgb(85, 187, 138);
border-radius:8px;
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
